src/transformers/models/albert/modeling_albert.py

```diff
@@ -1088,7 +1088,12 @@ def __init__(self, config):
         self.num_labels = config.num_labels
 
         self.albert = AlbertModel(config, add_pooling_layer=False)
-        self.dropout = nn.Dropout(config.hidden_dropout_prob)
+        classifier_dropout_prob = (
+            config.classifier_dropout_prob
+            if config.classifier_dropout_prob is not None
+            else config.hidden_dropout_prob
```

Comment:
I am against this behavior - it's a breaking change and it will create inconsistency and hurt the reproducibility of previous research code since the same config will give different results. The expected behavior is to set it as `0` when no `config.classifier_dropout_prob` is provided.